includes/class-wp-job-manager-data-exporter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Defines a class to handle the user data export
+ *
+ * @package wp-job-manager
+ * @since 1.31.1
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! class_exists( 'WP_Job_Manager_Data_Exporter' ) ) {
+	/**
+	 * Handles the user data export.
+	 *
+	 * @package
+	 * @since
+	 */
+	class WP_Job_Manager_Data_Exporter {
+		/**
+		 * Register the user data exporter method
+		 *
+		 * @param array $exporters The exporter array.
+		 * @return array $exporters The exporter array.
+		 */
+		public function register_wpjm_user_data_exporter( $exporters ) {
```

Comment:
This needs to be a static function as it is being hooked with a static reference.